felix/config/config_params_test.go

```diff
@@ -504,6 +504,11 @@ var _ = DescribeTable("Config parsing",
 		map[string]time.Duration{"a": 0}, false),
 	// Not a required parameter so a bad value is translated to nil:
 	Entry("HealthTimeoutOverrides non-duration", "HealthTimeoutOverrides", "foo=bar", map[string]time.Duration(nil), false),
+
+	Entry("BPFForceTrackPacketsFromIfaces", "BPFForceTrackPacketsFromIfaces", "", []string{"docker+"}),
+	Entry("BPFForceTrackPacketsFromIfaces", "BPFForceTrackPacketsFromIfaces", "docker0", []string{"docker0"}),
+	Entry("BPFForceTrackPacketsFromIfaces", "BPFForceTrackPacketsFromIfaces", "docker+", []string{"docker+"}),
+	Entry("BPFForceTrackPacketsFromIfaces", "BPFForceTrackPacketsFromIfaces", "docker0,docker1", []string{"docker0", "docker1"}),
```

Comment:
Please add an entry that fails validation.

Reply:
Thanks @fasaxc I as per request I am trying to add an entry that fails validation something like
```Entry("BPFForceTrackPacketsFromIfaces", "BPFForceTrackPacketsFromIfaces", "cali@123", []string{"docker+"}, true),```
But in its current form the test will not pass because the config params `die-on-failure` was removed from `BPFForceTrackPacketsFromIfaces` as suggested prior.  I see when this is the case the expected Error will never be true as the err is reverted to nil and the updated parameter will revert to its default value i.e. `docker+` in this case.  Therefore, I can add the following test instead
```Entry("BPFForceTrackPacketsFromIfaces", "BPFForceTrackPacketsFromIfaces", "cali@123", []string{"docker+"}),```
 i.e. expected Error false but pls let me know if you would like the config params `die-on-failure` validation to be re-instated - thanks

Reply:
SGTM

Reply:
Align config params test with params type test and include valid and invalid use cases